edge-modules/DirectMethodSender/Settings.cs

```diff
@@ -28,7 +28,12 @@ public class Settings
                     configuration.GetValue<TimeSpan>("DirectMethodDelay", TimeSpan.FromSeconds(5)),
                     Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")),
                     configuration.GetValue<InvocationSource>("InvocationSource", InvocationSource.Local),
```

Comment:
>InvocationSource [](start = 81, length = 16)

Is InvocationSource used in other modules or why is it it the module util?

Reply:
I was thinking that other module may use it.